cmd/blockchaincmd/add_validator.go

```diff
@@ -124,6 +126,8 @@ Testnet or Mainnet.`,
 	cmd.Flags().StringVar(&nodeEndpoint, "node-endpoint", "", "gather node id/bls from publicly available avalanchego apis on the given endpoint")
 	cmd.Flags().Uint64Var(&weight, validatorWeightFlag, uint64(constants.DefaultStakeWeight), "set the weight of the validator")
 	cmd.Flags().StringVar(&validatorManagerOwner, "validator-manager-owner", "", "force using this address to issue transactions to the validator manager")
+	stakerPrivateKeyFlags.SetFlagNames("staker-private-key", "staker-key", "staker-genesis-key")
```

Comment:
we shouldnt have multiple key names for the same thing IMO. One key name is easier to users.
maybe just staker key

Reply:
also might not be secure in case users are using CLI in backend to expose private key, maybe tell users to first create the key through key create and then using the key name as input

Reply:
I believe it is of convenience to provide options to input a key name, or just directly the private key, or better genesis key, probably this is for another PR or discussion as there are many places where this options are received

Reply:
, probably this is for another PR or discussion as there are many places where this options are accepted.